src/components/dashboard/DashboardWrapper/index.jsx

```diff
@@ -0,0 +1,90 @@
+import { ReactComponent as CreateIcon } from 'assets/icons/dashboard-create.svg';
+import dropdownIcon1 from 'assets/icons/dashboard-dropdown-icon1.svg';
+import dropdownIcon2 from 'assets/icons/dashboard-dropdown-icon2.svg';
+import { ReactComponent as MoneyIcon } from 'assets/icons/dashboard-money.svg';
+import { ReactComponent as TeamIcon } from 'assets/icons/dashboard-people-team.svg';
+import DropdownMenu from 'components/shared/DropdownMenu';
+import Tabs from 'components/shared/Tabs';
+import React from 'react';
+import { incomingInvitations, offersData, transactionsData } from '../data';
+import InvitationsTable from '../InvitationsTable';
+import OffersTable from '../OffersTable';
+import TransactionsTable from '../TransactionsTable';
+import {
+  DashboardCard,
+  DashboardCardsContainer,
+  DashboardCardText,
+  DashboardContainer,
+  DashboardHeading,
+  DashboardTitle,
+  DropdownMenusContainer,
+  TransactionsContainer,
+} from './dashboard-wrapper.styles';
+
+const cardsInfo = [
+  {
+    Icon: TeamIcon,
+    text: 'Start New Selling /Buying Transactions',
+  },
+  {
+    Icon: MoneyIcon,
+    text: 'Invite Other Parties to my Transactions',
+  },
+  {
+    Icon: CreateIcon,
+    text: 'Create Listing',
+  },
+];
+
+/**
+ * Dashboard Wrapper component
+ *
+ * @return {JSX.Element}
+ */
+function DashboardWrapper() {
+  return (
+    <DashboardContainer>
+      <DashboardTitle>Hi Belle Doe,</DashboardTitle>
+
+      <DashboardCardsContainer>
+        {cardsInfo.map(({ Icon, text }, index) => (
+          <DashboardCard key={index.toString()}>
+            <Icon />
+            <DashboardCardText>{text}</DashboardCardText>
+          </DashboardCard>
+        ))}
+      </DashboardCardsContainer>
+
+      <DashboardHeading>Offers</DashboardHeading>
+      <OffersTable data={offersData} />
+
+      <TransactionsContainer>
+        <DashboardHeading>My Current Transactions</DashboardHeading>
+        <DropdownMenusContainer>
+          <DropdownMenu name="status" leftIcon={dropdownIcon1}>
+            <option value="">Status</option>
+            <option value="1">status 1</option>
+            <option value="1">status 3</option>
+          </DropdownMenu>
+          <DropdownMenu name="role" leftIcon={dropdownIcon2}>
```

Comment:
Names are not clear here, please fix the above too if possible. 🙏 
```suggestion
          <DropdownMenu name="role" leftIcon={roleDropDownIcon}>
```